doorman/src/main.cpp

```diff
@@ -1,5 +1,7 @@
 #include <Arduino.h>
 #include <DNSServer.h>
+#include <random>
+#include <chrono>
 
```

Comment:
Is the Chrono import used? I think delay is coming from Arduino.h

Reply:
```c++
std::srand(std::time(0));
```

Reply:
Why don't you use millis()? It's available and should not introduce extra deps.